app/javascript/components/App.js

```diff
@@ -0,0 +1,45 @@
+import React, { useState } from 'react';
+import { Route, Routes } from 'react-router-dom';
+import Stack from 'react-bootstrap/Stack';
+import Container from 'react-bootstrap/Container';
+import Home from '../pages/Home';
+import Reservation from '../pages/Reservation';
+import NewCourse from '../pages/NewCourse';
+import DeleteCourse from '../pages/DeleteCourse';
+import NewReservation from '../pages/NewReservation';
+import SideNavbar from './SideNavbar';
+import Login from '../pages/Login';
+import Detail from '../pages/Details';
+import Header from './Header';
+import Profile from '../pages/Profile';
+
+function App() {
```

Comment:
- [ ]  Please, kindly refactor this and all other functions in your project (5) to ES6 arrow functions, so your code complies with the latest standards.

Refactoring is easy:

```javascript
const App = () => {
  // ...
}
```

## Explanation

The `function` keyword has been soft deprecated in almost all the software industry in favor of ES6 arrow functions. This means, almost nobody uses it anymore, so much that it's considered a non-best practice.

You can do a search in Vscode for the "function" word, and correct as needed (I found 5).  Shouldn't take you more than 5 min so don't worry:

![image](https://github.com/IndieCoderMM/e-learning-academy/assets/57421823/98c2e09d-8053-4b37-9d39-7a0ffea6c870)

That way, your code will meet the latest standards 💪

Reply:
Thanks so much for your very detailed and insightful review. 🙏 
We truly appreciate it and we'll be implementing all the required changes.